### PR TITLE
nit: CycleTracker - Secure connection - minor typos and rewordings

### PR DESCRIPTION
**Pull request by SphinxKnight**
Pushing some fixes detected while working on the French version.

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ There are several {{glossary("IDE")}} extensions and programming-language specif
...
 
-Note that the Preview on Web Server extension uses browserSync. When your development environment is started by this extension, `localhost:3001` provides an overview of the current server environment.
+> **Note:** that the Preview on Web Server extension uses browsersync. When your development environment is started by this extension, `localhost:3001` provides a user interface for Browsersync, providing an overview of the current server environment.
```
Comment (bsmth) [inline files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:44]: Typo?


```suggestion
> **Note:** that the Preview on Web Server extension uses Browsersync. When your development environment is started by this extension, `localhost:3001` provides a user interface for Browsersync, providing an overview of the current server environment.
```

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ Service workers, and therefore PWAs, are [restricted to secure contexts](/en-US/
 
 In the previous section, we used HTML and CSS to create the shell of our period tracking application. In this section, we'll open the CycleTracker static content in a browser, view the content from a locally started development environment, and view the content on a remote, secure server.
 
-## Viewing with the files:// protocol
+## Viewing with the `file://` protocol
```
Comment (bsmth) [inline files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:16]: Just FYI backticks in headings is controversial and a pet peeve of some contributors.

Comment (SphinxKnight) [reply files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:16]: I thought consensus had been decided in https://github.com/mdn/mdn-community/discussions/271#discussioncomment-4225841

Comment (bsmth) [reply files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:16]: Ah, very good, thanks for finding the conversation. I'm resolving the comment, in that case 👍🏻

**Review by bsmth — APPROVED** [review]
Thanks @SphinxKnight - just leaving a +1, not merging now in case @hamishwillee would like to chime in, also.

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:16]: FWIW IMO consensus was that code markup in headings is allowed, not that it is mandated. This case in particular is one I am happy with.

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 3]
```diff
@@ -75,19 +75,19 @@ To get the added features of PWAs, including single click installation, a standa
 
 When officially publishing a PWA, you will likely want to invest in a [domain name and web hosting](/en-US/docs/Learn/Common_questions/Tools_and_setup/How_much_does_it_cost#hosting). For open source projects, where developers can learn from the codebase and even contribute back to the project, you can host your progress on [GitHub Pages](https://pages.github.com/).
 
-## GitHub pages
+## GitHub Pages
```
Comment (hamishwillee) [inline files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:78]: Good catch on the proper noun. I bet this one comes up again.

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ There are several {{glossary("IDE")}} extensions and programming-language specif
...
 
-Note that the Preview on Web Server extension uses browserSync. When your development environment is started by this extension, `localhost:3001` provides an overview of the current server environment.
+> **Note:** that the Preview on Web Server extension uses Browsersync. When your development environment is started by this extension, `localhost:3001` provides a user interface for Browsersync, providing an overview of the current server environment.
```
Comment (hamishwillee) [inline files/en-us/web/progressive_web_apps/tutorials/cycletracker/secure_connection/index.md:44]: I don't know what Browsersync is. It should be defined. However not blocker of this PR.

**Review by hamishwillee — APPROVED** [review]
Looks like a good set of fixes to me. More than just nits - particularly `file://` vs `files`.

EDIT. Where are my manners. Thank you!

**Pull request merged**